src/util/createStore.js

```diff
@@ -19,22 +23,35 @@ const noopMiddleware = store => next => action => next(action);
  * @param {Function} middleware (optional) any app-specific middleware that
  *   should be applied to the store
  */
-function finalCreateStore(routes, reducer, initialState=null, middleware=[]) {
-	// **All** middleware gets added here
-	const middlewareToApply = [
-		getPlatformMiddleware(routes),
-		typeof window !== 'undefined' && window.mupDevTools ? window.mupDevTools() : noopMiddleware,
+function createEnhancedStore(router, reducer, initialState=null, middleware=[], enhancers=[]) {
```

Comment:
A `store` that has been created with [store enhancers](https://github.com/reactjs/redux/blob/master/docs/Glossary.md#store-enhancer)